src/signer_node.rs

```diff
@@ -178,6 +179,11 @@ impl<T: TapyrusApi, C: ConnectionManager> SignerNode<T, C> {
         self.round_timer.start().unwrap();
         // get error_handler that is for catch error within connection_manager.
         let connection_manager_error_handler = self.connection_manager.error_handler();
+
+        // Add signal handler
+        let unix_stop_signal_handler =
+            set_stop_signal_handler().expect("Failed to register signal handler.");
+
```

Comment:
In this scope, no log is output before connecting redis. Is it better to control on the main function in `node.rs`?

Reply:
it looks good. i will change.